torch/quantization/QConfig.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from collections import namedtuple
+
+class QOptions(object):
```

Comment:
I'd drop this method all together. AFAIU, it's only used to pass arguments to constructor of observer. You can just allow to pass arbitrary arguments there (including dtype/qscheme directly).

Reply:
OK, yeah I feel this is not really used as well.